### PR TITLE
Update starters to include __main__.py for 0.17.2

### PR DESCRIPTION
**Pull request by limdauto**
In 0.17.2, we have followed Python standard to use `__main__.py` as the entrypoint for running the Kedro project in package mode.

### CODE REVIEW

**Review by AntonyMilneQB — APPROVED** [review]

File: astro-iris/{{ cookiecutter.repo_name }}/src/setup.py [hunk 1]
```diff
@@ -52,16 +52,13 @@
     install_requires=requires,
     extras_require={
         "docs": [
-            "sphinx>=1.6.3, <2.0",
-            "sphinx_rtd_theme==0.4.1",
-            "docutils",
-            "nbsphinx==0.3.4",
+            "sphinx~=3.4.3",
+            "sphinx_rtd_theme==0.5.1",
```
Comment (lorenabalan) [inline astro-iris/{{ cookiecutter.repo_name }}/src/setup.py:56]:
```suggestion
            "sphinx_rtd_theme==0.5.1",
```

On kedro we have `0.4.1` - is it a typo here or are you just wanting to use latest versions?

**Review by lorenabalan — APPROVED** [review]
LGTM

File: astro-iris/{{ cookiecutter.repo_name }}/src/setup.py (hunk 1 above)
Comment (limdauto) [reply astro-iris/{{ cookiecutter.repo_name }}/src/setup.py:56]: Hmm I copied this from Kedro. Let me double check.

Comment (limdauto) [reply astro-iris/{{ cookiecutter.repo_name }}/src/setup.py:56]: Yea I copied this from the main repo's starter: https://github.com/quantumblacklabs/private-kedro/blob/master/kedro/templates/project/%7B%7B%20cookiecutter.repo_name%20%7D%7D/src/setup.py#L56

**Pull request merged**